.github/workflows/release.yml

```diff
@@ -69,3 +69,23 @@ jobs:
           artifacts: "./system.json, ./system.zip"
           tag: ${{ github.event.release.tag_name }}
           body: ${{ github.event.release.body }}
+      # Post to Package Release API
+      - name: Post to Package Release API
+        run: |
+          curl --request POST \
+          --url https://api.foundryvtt.com/_api/packages/release_version \
+          --header 'Authorization: ${{ secrets.FVTT_PACKAGE_RELEASE_TOKEN }}' \
+          --header 'Content-Type: application/json' \
+          --data '{
+            "id": "ose",
+            "release": {
+              "version": "${{ github.event.release.tag_name }}",
+              "manifest": "https://github.com/${{github.repository}}/releases/download/${{github.event.release.tag_name}}/system.json",
+              "notes": "https://github.com/https://github.com/${{github.repository}}/releases/tag/${{github.event.release.tag_name}}",
+              "compatibility": {
+                "minimum": "11",
+                "verified": "11",
+                "maximum": "11"
```

Comment:
It'd be cool if these could be dynamically set somehow so we can leave our CI file alone when a new Foundry version comes out.

Reply:
You make a good point. This will be fixed before I merge this